content/applications/sales/point_of_sale/pricing/pricelists.rst

```diff
@@ -2,45 +2,122 @@
 Pricelists
 ==========
 
-You probably know the concept of happy hour: during a certain period of
-time, the barman gives a discount on some drinks (usually 50% off or a
-buy one get one free). When the period is over, prices go back to
-normal. But how does that relate with Odoo?
-
-In Odoo, you can set up happy hours. It’s one of the many possible uses
-of *Pricelists*. Those *Pricelists* allow the creation of
-multiple prices for the same product: a regular one and a special one
-for happy hours. Available in the *PoS* app, those are really
-convenient.
-
-Set up Pricelists
+The pricelist feature allows you to adjust product prices based on predefined criteria.
+
+It can be used for a designated duration, like a temporary sale on products within a particular PoS
+category, for specific users, or as a reward when customers meet certain conditions, such as a
+minimum purchase quantity or amount.
+
+Configuration
+=============
+
+To **activate** the pricelist feature, access the :ref:`general PoS app's settings
+<configuration/settings>` and enable :guilabel:`Pricelists` in the :guilabel:`Pricing` section.
+
+Then, select the :ref:`type of pricelist <pricelists/create>` by checking :guilabel:`Multiple prices
+per product` or :guilabel:`Advanced price rules (discounts, formulas)`, save, and click
+:guilabel:`Pricelists` to create new ones or modify existing ones.
```

Comment:
I agree with @auva-odoo 's comment to better explain the difference between the two types (multiple / advanced rules), or maybe add an example section?

I wanted to do it but I'm afraid I have to leave, so we'll see on Monday morning :)